providers/bandwidth/src/lib/bandwidth.provider.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { BandwidthSmsProvider } from './bandwidth.provider';
+
+test('should trigger BandwidthSmsProvider library correctly', async () => {
+  const provider = new BandwidthSmsProvider({
+    username: '<your-bandwidth-username>',
+    password: '<your-bandwidth-password>',
+    accountId: '<your-bandwidth-accountId>',
+  });
+
+  const spy = jest
+    .spyOn(provider, 'sendMessage')
```

Comment:
instead of mocking the `sendMessage` can you try and mock the `createMessage` of the controller?

Reply:
Yes, please try. let me know if you need any help

Reply:
Done 👍. Please let me know if there's anything else required from my end